jena-arq/Grammar/sparql_11.jj

```diff
@@ -1318,6 +1318,8 @@ Expr Aggregate() : { Aggregator agg = null ; String sep = null ;
     { agg = AggregatorFactory.createMax(distinct, expr) ; }
   | t = <AVG> <LPAREN> ( <DISTINCT> { distinct = true ; } )? expr = Expression() <RPAREN>
     { agg = AggregatorFactory.createAvg(distinct, expr) ; }
+  | t = <MEDIAN> <LPAREN> ( <DISTINCT> { distinct = true ; } )? expr = Expression() <RPAREN>
```

Comment:
Shouldn’t be included in the standard SPARQL 1.1 grammar, if properly #ifdef’d in `master.jj` this will be resolved when you regenerate the grammars

Reply:
thx rvesse, made the grammar changes in the fork. will this automatically update the PR as well?

Reply:
Adding a commit to your branch in the repo from which you made the PR (looks like your own repo) will add the commit to the PR as well. The PR is essentially defined as being between the tips of the two branches between which you initiated it, in this case `apache:master` from `neumarcx:master`.